pallets/did/src/did_details.rs

```diff
@@ -612,28 +637,51 @@ impl<T: Config> DidDetails<T> {
 	}
 }
 
+pub(crate) type DidNewKeyAgreementKeys<T> = BoundedBTreeSet<DidEncryptionKey, <T as Config>::MaxNewKeyAgreementKeys>;
+pub(crate) type DidTotalKeyAgreementKeys<T> = BoundedBTreeSet<KeyIdOf<T>, <T as Config>::MaxTotalKeyAgreementKeys>;
```

Comment:
I think the `total` in `MaxTotalKeyAgreementKeys` is just there to stress that it's the sum?

```suggestion
pub(crate) type DidKeyAgreementKeys<T> = BoundedBTreeSet<KeyIdOf<T>, <T as Config>::MaxTotalKeyAgreementKeys>;
```